### PR TITLE
Use <NotificationModal /> to report player problems

### LINKED ISSUE #1409: Convert PlayerProblemModal to FluentUI

### PR DESCRIPTION
**Pull request by 2metres**
Replace the `<PlayerProblemModal />` with the `<NotificationModal />` given that they are basically the same.

Closes #1409

### CODE REVIEW

File: packages/studio-base/src/components/NotificationModal.tsx [hunk 1]
```diff
@@ -45,68 +22,47 @@ export default function NotificationModal({
...
-              marginLeft: theme.spacing.s1,
-              marginRight: theme.spacing.s1,
-              height: theme.spacing.l2,
-              lineHeight: theme.spacing.l2,
-              textAlign: "center",
-              flexShrink: 0,
+            field: {
+              color: theme.semanticColors.bodyText,
+              fontSize: "81.25%",
+              fontFamily: "'Ubuntu Mono', Menlo, Monaco, Courier, monospace !important",
```
Comment (defunctzombie) [inline packages/studio-base/src/components/NotificationModal.tsx:48]: should we be overriding the font here or using font semantics from the theme?

File: packages/studio-base/src/components/NotificationModal.tsx [hunk 2]
```diff
@@ -45,68 +22,47 @@ export default function NotificationModal({
...
-              verticalAlign: "top",
-              marginLeft: theme.spacing.s1,
-              marginRight: theme.spacing.s1,
-              height: theme.spacing.l2,
-              lineHeight: theme.spacing.l2,
-              textAlign: "center",
-              flexShrink: 0,
+            field: {
+              color: theme.semanticColors.bodyText,
+              fontSize: "81.25%",
```
Comment (defunctzombie) [inline packages/studio-base/src/components/NotificationModal.tsx:47]: Thats a rather specific size - can we use theme sizing?

**Comment by defunctzombie** [comment]
👍

File: packages/studio-base/src/components/NotificationModal.tsx (hunk 1 above)
Comment (2metres) [reply packages/studio-base/src/components/NotificationModal.tsx:48]: Ah yep, good catch.

**Pull request merged**